docs/source/cli.rst

```diff
@@ -31,7 +31,7 @@ Read it
     optional arguments:
       -h, --help            show this help message and exit
       --pattern PATH, -p PATH
-                            paths to pattern definition files
+                            paths to pattern definition files or modules
```

Comment:
@ajm188 Honestly, if we're only allowing modules that are on `sys.path`, I'm not sure why we're allowing path-like inputs in the first place. I would just enforce only something that can directly be passed to `importlib.import_module` (or `__import__`, which is basically equivalent but discouraged).